gh-release: add page (#4773)
tldr-pages/tldr: pages/common/gh-release.md

> # gh-release

Reviewer: the page header is without `-` because should be exact to the subcommand:
```suggestion
# gh release
```

> > Manage GitHub pull requests from the command line.

Reviewer: This subcommand is specific for releases
```suggestion
> Manage GitHub releases from the command line.
```

> > More information: <https://cli.github.com/manual/gh_release_delete>.

Reviewer: ```suggestion
> More information: <https://cli.github.com/manual/gh_release>.
```

> - List GitHub Releases in a repository (Default limit 30):

Reviewer: Maybe we can say `GitHub repository` in the first example, but I think repeat `GitHub` in every example is noisy
```suggestion
- List releases in a GitHub repository, limit 30 items:
```

Reviewer: Otherwise:
```suggestion
- List the last 30 releases in a GitHub repository:
```

`gh release list`

> - View a GitHub Release:

Reviewer: ```suggestion
- Display information about a specific release:
```

`gh release view {{tag}}`

> - Create a new GitHub Release:

Reviewer: ```suggestion
- Create a new release:
```

`gh release create {{tag}}`

> - Delete a GitHub Release:

Reviewer: ```suggestion
- Delete a specific release:
```

# gh release

> Manage GitHub releases from the command line.
> More information: <https://cli.github.com/manual/gh_release>.

> - List releases in a GitHub repository, limit 30 items:

Reviewer: ```suggestion
- List releases in a GitHub repository, limited to 30 items:
```

- Delete a specific release:

`gh release delete {{tag}}`

- Download assets from a specific release:

`gh release download {{tag}}`

- Upload assets to a specific release:

> `gh release upload {{tag}} {{file1 file2 ...}}`

Reviewer: ```suggestion
`gh release upload {{tag}} {{path/to/file1}} {{path/to/file2}}`
```
